indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -282,6 +283,30 @@ private void extractSubjects(Element mods, IndexDocumentBean idb) {
 
     }
 
+    private void extractOtherSubjects(Element mods, IndexDocumentBean idb) {
+        List<Element> otherSubjectEls = mods.getChildren("subject", JDOMNamespaceUtil.MODS_V3_NS);
+        List<String> otherSubjects = new ArrayList<>();
+        if (!otherSubjectEls.isEmpty()) {
+            for (Element otherSubjectObj : otherSubjectEls) {
+                List<Element> subjectParts = otherSubjectObj.getChildren();
+                for (Element otherSubject : subjectParts) {
+                    String otherSubjectName = otherSubject.getName();
+                    if (!otherSubject.getChildren().isEmpty() ||
```

Comment:
Turns out we will want to get the values from all the children rather than skipping subjects with children, so I guess we'll need a loop when children isn't empty. Anna is getting some examples for the test.